README.md

```diff
@@ -2,8 +2,56 @@
 
 # Alloy
 
-Alloy is the code name for the Adobe Experience Platform Web SDK. It allows for streaming data into the platform, syncing identities, personalizing content, and more.
+Alloy is the code name for the Adobe Experience Platform Web SDK. It allows for recording events into the Adobe Experience Platform, syncing identities, personalizing content, and more.
 
 For documentation on how to use Alloy, please see the [user documentation](https://adobe.ly/36dGGp6).
 
 For documentation on how to contribute to Alloy, please see the [developer documentation](https://github.com/adobe/alloy/wiki).
+
+## Installation
+
+There are three supported ways to use Alloy.
+  1. Using Adobe Launch, install the Adobe Experience Platform Web SDK Extension.
+  2. Use the pre-built, minified version available via a CDN. You could also self-host this version.
+  3. Use the NPM library which exports an ES6 module.
+
+### Using the launch extension
+
+For documentation on the AEP Web SDK Launch Extension see the [launch documentation](https://docs.adobe.com/content/help/en/launch/using/extensions-ref/adobe-extension/aep-extension/overview.html)
+
+### Using the stand alone version
+
+```html
+<script>
+  !function(n,o){o.forEach(function(o){n[o]||((n.__alloyNS=n.__alloyNS||
+  []).push(o),n[o]=function(){var u=arguments;return new Promise(
+  function(i,l){n[o].q.push([i,l,u])})},n[o].q=[])})}
+  (window,["alloy"]);
+</script>
+<script src="https://cdn1.adoberesources.net/alloy/2.2.0/alloy.min.js" async></script>
+<script>
+  window.alloy("config", { ... });
+  window.alloy("sendEvent", { ... });
+</script>
+```
+
+### Using the NPM library
+
+```bash
+npm install @adobe/alloy
+```
+
+Using the library:
+
+```javascript
+import { baseCode, core } from "@adobe/alloy";
+baseCode(["alloy"]);   // creates the window.alloy function
+core();                // runs
+
+window.alloy("config", { ... });
+window.alloy("sendEvent", { ... });
+```
+
+The ES6 exports are exposed via the NPM "module" property, so you may need to adjust your rollup system to look for the module property.
```

Comment:
Couple questions here: 

(1) Are you aware of any bundlers that don't support `module` by default? I'm aware it isn't standard, but I think it's been supported by default for 2+ years on the major bundlers. If so, this line may not be providing value and just introducing cognitive load, but who knows. If you keep it, it might be better to say something like "are exposed by the NPM package via the `module` field of its package.json, so you may...". If I'm a consumer having problems, that might give me more context for googling. 🤷‍♂️ 


(2) I'm trying to think through why we're creating `main.js` and `modules.js` as bundles. Normally what I've seen is that you would just run all the source code files through babel and dump all the resulting files into, say, an `es5` directory, then point the `main` field in package.json to `es5/index.js`. For the es6 variant, you may just be able to point the `module` field in package.json to `src/index.js`.

One of the benefits of not generating bundles for the npm package consumers is that if the consuming application already has a dependency that matches one of our dependencies, the final application bundle only needs to include the dependency code once. Currently, we only have a couple dependencies, so that chance is somewhat low, but that may grow over time

One effect of taking this approach is that the extension will need to do some bundling before getting packaged and uploaded. But, I believe this would mean that you would be able to move the Launch core module dependencies from `peerDependencies` back to `dependencies` in this repo. When you do the bundling for the extension, I believe you'd just mark Launch core module dependencies as "externals" so that Launch can provide them instead of bundling them in from `node_modules`.

Sorry, I should've thought through this during my previous review as it definitely affects what I was saying in https://github.com/adobe/alloy/pull/632/files#r518529523.